infrastructure/debugging-multi-layered-docker-deployment.md

```
@@ -0,0 +1,304 @@
+docker network inspect caddy_net | jq '.[]?.Containers | to_entries[] | {name: .value.Name, ipv4: .value.IPv4Address}'
+```
+-   Result showed `oullin_proxy_prod`, `web_caddy_prod`, `oullin_api` on `172.19.0.0/16`.
+
+
+### B) Fix the mTLS trust chain (web → API)
+
+-   Rotate/regenerate CA and client leaf on the API side.
+-   **Sync** `ca.pem`, `client.pem`, `client.key` into `web_caddy_prod` (mounted `caddy/mtls:ro`).
+-   Test from a web container:
+
+### B) Fix the mTLS trust chain (web → API)
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)